src/shipit_static_analysis/tests/test_issues.py

```diff
@@ -96,3 +90,25 @@ def __init__(self, **kwargs):
     paths = [i.path for i in diff]
     assert 'lib.h' in paths
     assert 'another.cpp' in paths
+
+
+def test_lines_hash(test_cpp):
```

Comment:
I think we should add additional cases to cover all the cases we want to support
1) Moving lines in the file without changing their indentation -> same hash
2) Moving lines in the file changing their indentation -> same hash
...